cmd/ctr/commands/snapshots/snapshots.go

```diff
@@ -271,6 +281,121 @@ var removeCommand = cli.Command{
 	},
 }
 
+var createCommand = cli.Command{
+	Name:      "create",
+	Usage:     "create a new snapshot from an image",
+	ArgsUsage: "[flags] <key> <image name>",
+	Description: `Creates a new snapshot from a specified image.
+
+When you're done, use the remove command.
```

Comment:
We already have `prepareCommand` and `viewCommand` for creating snapshots. Would it make sense to extend those instead? This one seems like does the same, but offers more CLI flags.